### PR TITLE
Adding a Getting Started

### PR DESCRIPTION
**Pull request by GrahamMThomas**
When I first attempted to use Restacker I was totally lost and I realized it was because there was no Getting Started page. So I create one with my limited understanding to help those with no understanding. I also created a diagram hopefully explaining the relationship between the 4 main files. 

If something in my understanding is incorrect please tell me and I will fix it!

Graham

### CODE REVIEW

**Review by pbnj — CHANGES_REQUESTED** [review]
Great idea. Love the "Getting Started" document.
I'd like to request few minor changes with regards to wording to make it a bit clearer.

File: docs/05-GETTING_STARTED.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+There are 4 important files needed to make restacker work.
+
+###Restacker.yml
+
+First, you need to configure your `~/.restacker/restacker.yml` file.
+
+This file is what allows your command line to connect and access the correct account.
+
+Validation:
+Keep tweaking this file until you can run `restacker list` and get a list of all the stacks.
```
Comment (pbnj) [inline docs/05-GETTING_STARTED.md]: This line should read:

> `restacker list` should return a list of stacks for a particular account

File: docs/05-GETTING_STARTED.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+###Template.json
+
+Once this is complete, you need to get your hands on a template JSON file.
```
Comment (pbnj) [inline docs/05-GETTING_STARTED.md]: Remove "once this is complete". You can reference the CloudFormation sample template file that will reside in `infrastructure/cloudformation` (currently pending PR)

File: docs/05-GETTING_STARTED.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+###Parameters.yml
+
+After you have this you need to fill out your parameters.yml file. If your template has explanations for each field be sure to read them.
```
Comment (pbnj) [inline docs/05-GETTING_STARTED.md]: Remove "After you have this" - a bit superfluous.
Remove "If your template has explanations for each field be sure to read them" - not sure what you mean here.
You can reference the Parameter sample file that will reside in `infrastructure/parameters/` (currently in a pending PR)

File: docs/05-GETTING_STARTED.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+After you have this you need to fill out your parameters.yml file. If your template has explanations for each field be sure to read them. 
+
+This is the most diffcult step, while some of the parameters are not as critical as others, one mistyped letter and you will have no idea what the issue is because of the lack of error messages. 
+
+Validation:
+Once you can run `restacker deploy` and your stack returns Creation_Complete.
+
+###Userdata.sh
+
+After this, you may need to setup a userdata.sh file.
```
Comment (pbnj) [inline docs/05-GETTING_STARTED.md]: Remove "After this" - a bit superfluous.
You may reference the userdata sample file that will reside in `infrastructure/userdata/` (currently in a pending PR).

**Review by pbnj — CHANGES_REQUESTED** [review]
Update wording in Getting_Started.md document.

**Review by pbnj — APPROVED** [review]

File: docs/05-GETTING_STARTED.md (hunk 3 above)
Comment (GrahamMThomas) [reply docs/05-GETTING_STARTED.md]: In the template.json, there are descriptions for each field. These descriptions really helped me understand what to enter in each field.

**Comment by GrahamMThomas** [comment]
Changes made.

**Pull request merged**